cmd/metrics.go

```diff
@@ -260,6 +260,14 @@ func cacheMetricsPrometheus(ch chan<- prometheus.Metric) {
 		prometheus.CounterValue,
 		float64(cacheObjLayer.CacheStats().getBytesServed()),
 	)
+	ch <- prometheus.MustNewConstMetric(
+		prometheus.NewDesc(
+			prometheus.BuildFQName("cache", "gc", "counter"),
+			"Total number of GC triggers upon high cache usage on current MinIO instance",
+			nil, nil),
+		prometheus.CounterValue,
+		float64(cacheObjLayer.CacheStats().getGCCounter()),
```

Comment:
it might be useful to collect metrics on the total gc collected rather than just number of gc invocations.

Reply:
the intent is to show how many times GC was invoked doesn't matter which disk @poornas - since cache works as a whole 
there is lot more to do not sure what would be clearly valuable. 

I didn't want to add too much granularity as it may not be that useful